docs/source/initialization.md

```diff
@@ -37,8 +37,8 @@ The initializer for `RequestChainNetworkTransport` has several properties which
 
 - `interceptorProvider`: The interceptor provider to use when constructing chains for a request. See below for details on interceptor providers.
 - `endpointURL`: The GraphQL endpoint URL to use for all calls.
-- `additionalHeaders`: Any additional headers that should be automatically added to every request. Defaults to an empty dictionary.
-- `autoPersistQueries`: Pass `true` if [Automatic Persisted Queries](https://www.apollographql.com/docs/apollo-server/performance/apq/) should be used to send an operation's hash instead of the full operation body by default. **NOTE:** To use APQs, you need to make sure to generate your types with operation identifiers. In your Swift Script, make sure to pass a non-nil `operationIDsURL` to have this output. Due to this restriction, this option defaults to `false`.
+- `additionalHeaders`: Any additional headers that should be automatically added to **every** request, such as an API key or a language setting. Headers that should not be sent with every request should be configured through an interceptor. Defaults to an empty dictionary. 
```

Comment:
still think this should be interceptor. Would allow to avoid questions like "who has priority? additionalHeaders or Interceptor?"